src/plugins/Freereader/Freereader.java

```diff
@@ -75,12 +81,71 @@ public void terminate()
 		Logger.normal(this, pluginName + " terminated.");
 	}
 	
-	public String getString(String key) {
-		// TODO return a translated string
-		return key;
+	/**
+	 * This code is only used by FredPluginL10n...
```

Comment:
Oh, if there's that much president might as well remain consistent for now
then. It seems fine to leave it like that. I will test/merge this and add
it to Transifex when I next get a block of time for Freenet. Thanks!

On Mon, Mar 30, 2015, 6:36 AM nicolas hernandez <notifications@github.com>
wrote:

> In src/plugins/Freereader/Freereader.java
> <https://github.com/freenet/plugin-Freereader/pull/4#discussion_r27379839>
> :
>
> > @@ -75,12 +81,71 @@ public void terminate()
> >  		Logger.normal(this, pluginName + " terminated.");
> >  	}
> >  	
> > -	public String getString(String key) {
> > -		// TODO return a translated string
> > -		return key;
> > +	/**
> > +	 * This code is only used by FredPluginL10n...
>
> Simply specify than this call should not be used outside of l10n logic.
>
>    - WoT : no comment -
>    https://github.com/freenet/plugin-WebOfTrust/blob/master/src/plugins/WebOfTrust/WebOfTrust.java#L3832
>    - FlogHelper : same comment -
>    https://github.com/freenet/plugin-FlogHelper/blob/master/src/plugins/floghelper/FlogHelper.java#L435
>    - Freetalk : same comment -
>    https://github.com/freenet/plugin-Freetalk/blob/master/src/plugins/Freetalk/Freetalk.java#L783
>
> ... remove or not remove, change comment or leaving like this ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freenet/plugin-Freereader/pull/4/files#r27379839>.
>